.gitignore

```diff
@@ -2,6 +2,9 @@
 .vscode/
 /.favorites.json
 
+# WebStorm extension
```

Comment:
What is this?

Reply:
Will delete this, it should be my IDE auto added line.